src/js/tech/tech.js

```diff
@@ -371,6 +373,17 @@ class Tech extends Component {
   }
 
   /**
+   * Get remote htmltrackelements
+   *
+   * @returns {HTMLTrackElementList}
+   * @method remoteTextTrackEls
+   */
+  remoteTextTrackEls() {
+    this.remoteTextTrackEls_ = this.remoteTextTrackEls_ || new HTMLTrackElementList();
```

Comment:
We'll also probably want to go through the same shenanigans we've gone through with the text track list to have it work across tech changes. See https://github.com/videojs/video.js/pull/2391
This can, and probably should, be tackled only once we have everything else working and could probably event go out as a separate PR.